value.go

```diff
@@ -53,6 +53,15 @@ func Null(iso *Isolate) *Value {
 	return iso.null
 }
 
+func NewStringFromByteArray(iso *Isolate, val []byte) (*Value, error) {
```

Comment:
We will probably want the inverse to be supported as well (from the v8 value in JS back to Go)

Reply:
As mentioned in another comment, the reason behind implementing byte array to string is to overcome the challenges with null character in binary strings. I'm not sure for which scenario we would want to implement the reverse.

Reply:
When v8go converts a `*Value` to a Go `string` (e.g. with `.String()`), it truncates the string at the first null character it encounters.

Reply:
I added a testcase in value_test.go where i am validating the same. If you print val.String(), it doesnt truncate the data even though there is a 0 byte in the input array.
Having said that, do you have any specific test-case? I can look into that scenario.

Reply:
I guess we could introduce a `(v *Value) BinaryString() []byte` or `(v *Value) BinaryString() BinaryString` depending on [this outcome](https://github.com/rogchap/v8go/pull/266#discussion_r780675085).

Reply:
> When v8go converts a `*Value` to a Go `string` (e.g. with `.String()`), it truncates the string at the first null character it encounters.

Note that we're not trying to solve #192 here. We may do that in a separate PR.